akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -12,9 +12,13 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
 
   private lazy val serialization = SerializationExtension(system)
 
+  private val NoneSerialized = ContainerFormats.Some.newBuilder().build().toByteArray
```

Comment:
@patriknw - `None` is essentially the protobuf `Some` without `Payload` defined.